Makefile

```diff
@@ -169,6 +172,22 @@ update_crds: _set_ctx _update_crds _reset_ctx
 ### uninstall: remove namespace and all CRDs from cluster
 uninstall: _set_ctx _do_uninstall _reset_ctx
 
+update_devworkspace_api:
+	go mod edit --require github.com/devfile/kubernetes-api@$(DEVWORKSPACE_API_VERSION)
+	go mod download
+	go mod vendor
+
+### ...
+update_devworkspace_crds:
```

Comment:
The structure for the doc comments is
```
### <name of makefile rule>: <explanation of what it does
```
This is used to get the output of `make help`.
```suggestion
### update_devworkspace_api: update version of devworkspace_api in go.mod
update_devworkspace_api:
	go mod edit --require github.com/devfile/kubernetes-api@$(DEVWORKSPACE_API_VERSION)
	go mod download
	go mod vendor

### update_devworkspace_crds: download updated devworkspace_crd definitions
update_devworkspace_crds:
```
You will also need to change line 234 below to increase column width.
```
@sed -n 's/^### /    /p' $< | awk 'BEGIN { FS=":" } { printf "%-22s -%s\n", $$1, $$2 }'

to

@sed -n 's/^### /    /p' $< | awk 'BEGIN { FS=":" } { printf "%-30s -%s\n", $$1, $$2 }'
```